app/models/flood_risk_engine/address_search.rb

```diff
@@ -0,0 +1,5 @@
+module FloodRiskEngine
+  class AddressSearch < ActiveRecord::Base
```

Comment:
I don't see any clean up code. I was expecting there to be something in the LocalAuthorityAddressForm along the lines of the code below added to the end of `#assign_to_enrollment`:
```
enrollment.address_search.destroy if result
```